optimum/neuron/distributed/decoder_models.py

```diff
@@ -62,6 +66,8 @@ def _parallelize(
                 device=device,
             )
             block.mlp = GPTNeoParallelMLP.transform(model, block.mlp, device=device)
+        if parallelize_embeddings:
+            model = GPTNeoParallelCrossEntropy.transform(model, model, device=device)
```

Comment:
This parameter was added to every transform to make allow to load weights from a `safetensors` checkpoint directly to the desired device (`cpu` or `xla`). In practice it is not used for now, but it could be one day.